src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -1092,13 +1092,9 @@ const TorrentsTable = new Class({
         // ratio
         this.columns['ratio'].updateTd = function(td, row) {
             const ratio = this.getRowValue(row);
-            let html = null;
-            if (ratio == -1)
-                html = '∞';
-            else
-                html = (Math.floor(100 * ratio) / 100).toFixed(2); //Don't round up
-            td.set('html', html);
-            td.set('title', html);
+            const string = (ratio == -1) ? '∞' : tofixedPointString(ratio, 2);
```

Comment:
Please use triple equals (and then test it!)

Reply:
If you want me to change it you must be sure that it will work...

Reply:
I tested it quickly and it looks fine.